streams/delimiter_stream.ts

```diff
@@ -130,6 +131,8 @@ export class DelimiterStream extends TransformStream<Uint8Array, Uint8Array> {
               // for a suffix disposition this branch would mean that the previous
               // chunk ended with a full match but was not enqueued.
               chunkStart = inspectIndex;
+            } else {
+              chunkStart = 0;
```

Comment:
**comment:** This is actually unnecessary: The chunk start is necessarily already zero here but I decided to add this here for readability reasons.